src/view/MyFolderView.java

```diff
@@ -40,7 +67,17 @@ public MyFolderView(MyFolderViewModel myFolderViewModel, LogoutViewModel logoutV
         username = new JLabel();
 
         JPanel buttons = new JPanel();
-        logOut = new JButton(myFolderViewModel.LOGOUT_BUTTON_LABEL);
+        JButton logOut = new JButton(myFolderViewModel.LOGOUT_BUTTON_LABEL);
+        JButton openFolder = new JButton(myFolderViewModel.OPEN_BUTTON_LABEL);
```

Comment:
so does it mean this user are only allowed to have 1 folder?

Reply:
I have a for-loop below to address all folders